packages/utils/src/useTableTools/rowsBuilder.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { NoResultsTable } from '@redhat-cloud-services/frontend-components/TableToolsTable/NoResultsTable';
+
+const emptyRows = [
+  {
+    cells: [
+      {
+        title: () => <NoResultsTable />, // eslint-disable-line react/display-name
+        props: {
+          colSpan: 3,
```

Comment:
Should this value be hardcoded? What if table has 4 columns?

Reply:
The colSpan is now based on the columns.